classes/UpgradeTools/CoreUpgrader/CoreUpgrader80.php

```diff
@@ -100,4 +101,24 @@ protected function upgradeLanguage($lang)
 
         // TODO: Update AdminTranslationsController::addNewTabs to install tabs translated
     }
+
+    /**
+     * Ask the core to disable the modules not coming from PrestaShop.
+     */
+    protected function disableCustomModules()
+    {
+        try {
+            $bulkToggleModuleStatusCommand = new BulkToggleModuleStatusCommand(
```

Comment:
PHPstan says:

`Instantiated class                                                     
         PrestaShop\PrestaShop\Core\Domain\Module\Command\BulkToggleModuleStatusCommand not found`

does anyone know how this could be fixed ?  `BulkToggleModuleStatusCommand` is in the core of prestashop, not available in the auto-upgrade module.

Why doesn't PHPstan throw the same error for `GenerateThemeMailTemplatesCommand` on line 80 in this same file ? 🤔

Reply:
Il te manque pas un `\` devant le namespace ?

Reply:
as seen with @jolelievre , only the `latest` phpstan was having an issue, that was because `CoreUpgrader80` was not ignored on latest.